lib/device-sockets/ios/notification.ts

```diff
@@ -0,0 +1,47 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+export class IOSNotification implements IiOSNotification {
```

Comment:
This can safely be just a holder for constants, or even exported variables that can be imported via destructing.